.github/actions/update-getting-started/action.yml

```diff
@@ -0,0 +1,17 @@
+name: "Update getting started version"
+description: "Reusable action to update the getting started version"
+inputs:
+  version:
+    required: true
+    description: "Recommended SOCI version"
+
+runs:
+  using: composite
+  steps:
+    - name: "Update getting started version"
+      shell: bash
+      run: |
+        version=${{ inputs.version }}
+        version=${version/v/} # strip prefix v if not already stripped
+        sed -i -E "s/version=\"([0-9]+\.){2}[0-9]+\"/version=\"${version}\"/" docs/getting-started.md
```

Comment:
Sorry, I don't think I was clear before. This is the part that I think would be useful in a script.

For example, if need to update this script for some reason (e.g. we add another reference to the version, say a link to the specific release) then right now we can't test the change without pushing to a fork.

IMO the actions yaml should just move inputs from the github env into variables and then call scripts that do the actual work.

Reply:
@Kern-- , oh yeah I misunderstood. Pushed a new version to address this. PTAL. Good callout though, this simplifies what I wanted. Thanks!